<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background-image: url(:/Images/background); background-repeat: no-repeat;
}</string>
   </property>
   <widget class="QPushButton" name="registerBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>282</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: #FFFFFF;
    padding: 2px;
   	font: 17pt 'Times New Roman';
    border-width: 6px;
    border-radius: 10px;
	border-color: #009999;
	border-style: solid;
}
QPushButton:hover {
   	    background: #009999;
        position: absolute;
        top: 50%;
        left: 50%;
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="failTxt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background: rgba(255, 0, 0, 0);
	color: white;
   	font: 17pt 'Times New Roman';
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>232</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: #FFFFFF;
    padding: 2px;
   	font: 17pt 'Times New Roman';
    border-width: 6px;
    border-radius: 10px;
	border-color: #009999;
	border-style: solid;
}
QPushButton:hover {
   	    background: #009999;
        position: absolute;
        top: 50%;
        left: 50%;
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	 border-radius: 50px;
	border: 3px solid;
	border-color: #009999;
	color: white;
   	font: 17pt 'Times New Roman';
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameInput">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	 border-radius: 50px;
	border: 3px solid;
	border-color: #009999;
	color: white;
   	font: 17pt 'Times New Roman';
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>621</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-image: url(:/Images/loginLogo); background-repeat: no-repeat;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
